src/GitHub.Api/ApiClientConfiguration_User.cs

```diff
@@ -1,11 +1,13 @@
+﻿using System;
+
 namespace GitHub.Api
 {
-    public partial class ApiClient : IApiClient
+    static partial class ApiClientConfiguration
```

Comment:
It would be nice if rather than getting the `ClientId` / `ClientSecret` from source, it got if from an environment variable (`GITHUB_CLIENT_ID` or something). That way the developer wouldn't have to dirty their repo whenever they wanted to try something out (a hassle when changing branches). It could complain with a visible/helpful error when these variables aren't set.